ted/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -634,6 +654,20 @@ public void zeroRobotPosition()
     this.setRobotPosition(new Pose2d(0,0,Rotation2d.fromDegrees(0)));
   }
 
+  /**
+   * A method that updates the robot position with a vision measurement
+   * @param visionMeasurement the most recent vision measurement provided by vision subsystem
+   */
+  private void addVisionMeasurement(VisionMeasurement visionMeasurement){
+    // The wpilib matrix constructor requires sizes specified as Nat types. 
+    // trying to update the matrix dynamicaly leads to issues
+    Matrix<N3,N1> visionStdDev = (new MatBuilder<N3,N1>(Nat.N3(), Nat.N1())).fill(new double[]{0.9, 0.9, 0.9});
```

Comment:
MatBuilder is deprecated.  Find the proper replacement.

Reply:
replacement is here: https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/MatBuilder.html#fill(edu.wpi.first.math.Nat,edu.wpi.first.math.Nat,double...)

Reply:
?